packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -235,32 +222,6 @@ export class FrontendPluginImpl implements PluginImpl {
     );
   }
 
-  public async executeUserTask(func: Func, ctx: PluginContext): Promise<TeamsFxResult> {
```

Comment:
Just realized - is `executeUserTask` part of plugin interface? Totally removing this method or just let it do nothing?

Reply:
Both ok for me. If we do not need this interface now, just remove it.

Reply:
I addded `executeUserTask ` in PR: https://github.com/OfficeDev/TeamsFx/pull/2023
Before that frontend plugin does not contain the `executeUserTask ` lifecycle. @hund030 What do you think about it?